README.md

```diff
@@ -36,7 +36,7 @@ $ npm install
 $ PERCY_TOKEN=<YOUR_PROJECT_TOKEN> npm run test
 ```
 
-Option 2: In case you have already installed @percy/agent node module globally on your machine.
+Option 2: In case you have already installed @percy/cli node module globally on your machine.
```

Comment:
We recommend against installing `@percy/cli` globally. Since this is an example app, we know it's installed locally so we can use `npx` here. (`npx percy exec`) `npx` will look up the binary in the `node_modules` folder. Basically a shortcut to doing `node_modules/.bin/percy exec`